### ISSUE TITLE
Possible 2D heatmaps for videopose3D

### ISSUE THREAD

**Issue by Thevakumar-Luheerathan** [open-mmlab/mmpose#939]
I planned to use videopose3d provided by MMPOSE to my final year project. I have already ran the demo with HRNet (as it is). I just want to know what are the possible heatmaps (2D pose estimators instead of HRNet ) which can be used along with videopose3D? Can I use all configurations given in `mmpose/configs/body/2d_kpt_sview_rgb_img/topdown_heatmap` for better performance ?

If I can use all the 2D keypoint extractors, How is it possible? Because, we need train 3D lifter again based on the 2D pose estimator.(If I use detectron as 2D estimator, I should train videopose3D accordingly). That's what I thought. 

It would be much helpful in my FYP if you can give me clarification on this.
Thank you in advance .

**Comment by ly015** [comment]
Hi, thanks for being interested in MMPose, and hope you enjoy it in your project.

If you plan to implement a 2-stage 3D pose estimation model, it's necessary to keep the 2D keypoint definition consistent in both stages. For example, if you use a 3D lifter trained on Human3.6M data, you would need a 2D keypoint detector whose output is also in the Human3.6M definition. To do this, you can choose a 2D model trained on Human3.6M (we provided several models [here](https://github.com/open-mmlab/mmpose/tree/master/configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/h36m)), or manually convert the 2D output into the Human3.6M format (like in the demo). We would recommend the second approach because 2D models trained on COCO are more robust than Human3.6M (refer to #785.).

**Comment by Thevakumar-Luheerathan** [comment]
You guys are really awesome. Your work should be appreciable much. Thanks for answering quickly. I have another doubt. Can I use any other 2D models(instead of HRNet Can I use Alphapose, openpose and so on) with the given 3D model (`https://download.openmmlab.com/mmpose/body3d/videopose/videopose_h36m_243frames_fullconv_supervised_cpn_ft-88f5abbb_20210527.pth`) without further training? Or should I introduce a training step to the 3D model?

**Comment by ly015** [comment]
Yes, I think you can use any 2D keypoint model and do not need to re-train or finetune the 3D model, as long as the intermediate 2D keypoint definition is consistent, as mentioned above. On the other hand, if the accuracy is what matters in your project, a training step to the 3D model with the 2D output from the specific 2D model (instead of the 2D ground-truth) could possibly improve the accuracy of the 3D estimation.

**Comment by Thevakumar-Luheerathan** [comment]
Thank you.. I'll give a try..
